sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/plan/DefaultStepFactoryTest.java

```diff
@@ -239,6 +286,36 @@ private PodInstance getPodInstanceWithSameResourceSets() throws Exception {
         return new DefaultPodInstance(podSpec, 0);
     }
 
+    private PodInstance getPodInstanceWithFinishedTask() throws Exception {
```

Comment:
Technically, we should test ONCE and FINISH as well as they route similarly.

Oh to have table driven tests a la Go.